charger/ocpp.go

```diff
@@ -351,31 +355,28 @@ func (c *OCPP) setPeriod(current float64, phases int) error {
 	c.log.TRACE.Printf("current phases: %d, current current: %f", phases, current)
 
 	enabled, err := c.Enabled()
-	if err == nil && enabled {
-		err = c.setChargingProfile(c.connector, getTxChargingProfile(current, phases))
-	} else {
-		c.log.TRACE.Printf("no running transaction, so silently ommiting request to change charging parameters")
-	}
-
-	if err != nil {
-		err = fmt.Errorf("failed to set charging profile: %w", err)
+	if err == nil {
+		if enabled {
+			if err = c.setChargingProfile(c.connector, getTxChargingProfile(current, phases)); err != nil {
+				err = fmt.Errorf("failed to set charging profile: %w", err)
+			}
+		} else {
+			c.log.TRACE.Printf("no running transaction, ommitting request to change charging parameters")
+		}
 	}
 
 	return err
 }
 
-
 func getTxChargingProfile(current float64, phases int) *types.ChargingProfile {
-	if phases == 0 {
-		phases = 3
-	}
-
 	period := types.NewChargingSchedulePeriod(0, current)
-	period.NumberPhases = &phases
+	if phases != 0 {
+		period.NumberPhases = &phases
+	}
```

Comment:
Phases = 0 doesn't mean phases = 3?

Reply:
Not here. Auto is only available on the loadpoint.

Reply:
So phases = 0 means no initialization? Right?

Reply:
Right- "don't care" really